### ISSUE TITLE
Missing capability: Linking to filesystem files and internal links within the document

### ISSUE THREAD

**Issue by jeff9315** [mohzy83/NppMarkdownPanel#8]
As I stated earlier this morning, I LOVE your plugin. There are 2 types of links that don't get followed correctly. Hopefully, linking to files in the filesystem and to internal links in the document are both something you feel should be addressed:

(1)Internal links such as 
~~~
Jump to: [top](#top) ... [today](#today) ... [thisweek](#thisweek) ... [deferred](#deferred) ...[bottom](#bottom)
...
<a name="today"></a>
...
<a name="deferred"></a>
~~~
and also links to separate md files such as
~~~
* See [Comcast Issues](./Comcast.md)
~~~

**Comment by mohzy83** [comment]
Hi Jeff, 
you are right, internal links should work. It should be easy to fix that.
Currently only external links are supported. External links are opened in your default browser.
What is the desired behavior for linked md files? Preview them and also open the file in the Editor?

**Comment by jeff9315** [comment]
Yes, I think they should be opened in both the the Editor and the Preview Panel ... as a separate tab.

**Comment by czdiao** [comment]
Hi! Thanks a lot for making such a nice plugin!

In addition to the "open the links to separate md file", I am also wondering if it is possible to add the functionality to "open linked files with system default programs"? I use md file to manage some notes of my work. Quite often in the note I would refer to some pdf or xlsx files I worked on everyday. 

Thanks!

**Comment by xmatic** [comment]
> linking to files in the filesystem

I would like to add my support to this.

E.g., in Notepad++, if I add the following:

`[Salsa Pretzel Move](file:///C:\Users\user.name\Dropbox\Learn\Dance\SalsaPretzelMove.mp4)`

Then it creates a link in Notepad++ and I can double-click on this link, and it will directly open the video file in the default video handler app.

If I use another Markdown plugin (MarkdownViewer++), I see `Salsa Pretzel Move` as a link, and if I click on the link, it will directly open the video file in the default video handler app.

If I use NppMarkdownPanel, I see `Salsa Pretzel Move` as a link, but nothing happens if I click on the link. It would be nice if it could handle the click.

To @mohzy83 : awesome job on `NppMarkdownPanel`. Overall, it is my preferred Markdown plugin.

**Comment by mohzy83** [comment]
@jeff9315  plz check if the new version 0.9.0 resolves your issue https://github.com/mohzy83/NppMarkdownPanel/releases/tag/0.9.0
 Please make sure you use as renderer WebView2!

**Comment by user114145** [comment]
This does not seem to working yet. 
(V0.9, WebView2, Win10, admin rights)

Apart from that; great plugin! Really nice work.
